### DISCUSSION TITLE
LSPosed not installed

### DISCUSSION THREAD

**Discussion by HamzAnanS** [LSPosed/LSPosed#1904] [Q&A] [answered]
Zygisk LSPosed was successfully flashed but no promote popped up, no shortcut was created, even dial didn't work. I manually installed manager.apk and opened but it says "LSPosed not installed"

Device: Redmi 6A (Cactus)
Magisk v24.3
Zygisk LSPosed 1.8.2

**Comment by yujincheng08** [comment, marked as answer]
Need a full log to investigate.
